src/components/accounts/index.tsx

```diff
@@ -60,17 +63,22 @@ const AccountItem = ({ label, iconComponent, isFirst, hasAction, ...rest }: Acco
       cursor={hover && hasAction ? 'pointer' : 'unset'}
       bg={hover && hasAction ? 'ink.100' : 'white'}
       mt={isFirst ? 5 : 0}
-      px={2}
+      pl={2}
       transition={transition}
       {...bind}
       {...rest}
     >
       {iconComponent && iconComponent({ hover })}
       <Box overflow="hidden">
-        <Text display="inline-block" textAlign="left" textStyle="body.small.medium">
+        <Text display="inline-block" textAlign="left" textStyle="body.small.medium" {...loadingProps(!!loading)}>
           {label}
         </Text>
       </Box>
+      {loading && (
+        <Box position="absolute" right={4}>
+          <Spinner width={4} height={4} {...loadingProps(!!loading)} />
+        </Box>
+      )}
```

Comment:
I'm not positive we should do an absolutely positioned loader that is on the right side. Did you get a chance to test what happens with a very long name? Just curious as to how it will look when the loader is over the text in this situation.